docs/.vitepress/config.mts

```diff
@@ -134,8 +134,5 @@ export default defineConfig({
     }
   },
 
-  ignoreDeadLinks: [
-    './CODE-OF-CONDUCT',
-    './GOVERNANCE'
-  ]
+  ignoreDeadLinks: true
```

Comment:
For now i've decided to ignore all dead links, because we're still kind of WIP with these.